Sources/Mantis/Helpers/TransformRecord.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+
+enum TransformType {
+    case resetTransforms
+    case transform
+}
+
+class TransformRecord: NSObject {
+    
+    let transformType : TransformType!
+    
+    var actionName : String!
+        
+    let previousValues :  Dictionary<String, Any?>!
+    let currentValues : Dictionary<String, Any?>!
+    
+    var useCurrent : Bool! = true
+    
+    init(transformType: TransformType, actionName: String, previousValues: [String: Any?], currentValues: [String: Any?]) {
```

Comment:
Below is the property definitions I suggest.

* If you use `let` for a property and initialize it in the init(), no need to add `!`
* Use `let` if the property does not change like `actionName`

```Swift
    private let transformType: TransformType
    
    private let actionName: String
        
    private let previousValues:  Dictionary<String, Any?>
    private let currentValues: Dictionary<String, Any?>
    
    private var useCurrent: Bool = true
```

Reply:
changes made